src/System.Runtime.Caching/tests/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp-Windows_NT;
+      netfx;
```

Comment:
Does S.R.Caching not work on non-windows?

Reply:
Correct but @kkhurin tells me a port would be straightforward - replace the one pinvoke. Any interest?

Reply:
Oops I didn't notice you were @mconnew ...

Reply:
With the current cache memory management design the main missing *external* piece to fully support all platforms is #25617. Once this is implemented, the cache will still likely need some "adjustments" as the way of deriving "cache memory pressure" from "memory usage" is likely to be different on Linux (or even different on different versions of Linux). This invites further design decisions whether to let the cache "translate" memory readings into memory pressure itself or to delegate this to the user code (or have both the default algorithm and an opportunity to override it)... 

Until then the plan is to enable Linux (in fact all netstandard) support but the cache will throw PNSE if the corresponding memory settings are provided on the platforms where we don't have comprehensive memory info yet.